pkg/ratelimit/resource.go

```diff
@@ -140,10 +140,10 @@ func GatewayAcceptMQTTConnectionResource(remoteAddr string) Resource {
 	}
 }
 
-// GatewayUDPTrafficResource represents UDP gateway traffic from a remote IP address.
+// GatewayUDPTrafficResource represents UDP gateway traffic from a remote address.
 func GatewayUDPTrafficResource(addr *net.UDPAddr) Resource {
 	return &resource{
-		key:     fmt.Sprintf("gs:up:udp:ip:%s", addr.IP.String()),
+		key:     fmt.Sprintf("gs:up:udp:addr:%s", addr.String()),
```

Comment:
Doesn't this technically break config?

Reply:
This is key (identifying the resource), not class (identifying the limit). Also, TTES has been using this key the whole time for the past two years - it's just a backport which I've discovered while looking at the history of rate limiting.